pkgs/development/libraries/boost/generic.nix

```
@@ -1,10 +1,18 @@
-{ lib, stdenv, icu, expat, zlib, bzip2, python ? null, fixDarwinDylibNames, libiconv
+{ lib
+, stdenv
+, icu
+, expat
+, zlib
+, bzip2
+, python ? null
+, fixDarwinDylibNames
+, libiconv
 , boost-build
 , fetchpatch
 , which
-, toolset ? /**/ if stdenv.cc.isClang  then "clang"
-            else if stdenv.cc.isGNU    then "gcc"
-            else null
+, toolset ? /**/ if stdenv.cc.isClang then "clang"
```

```suggestion
, toolset ? if stdenv.cc.isClang then "clang"
```
If this is being formatted that comment no longer servers a purpose.

---

Yeah, unfortunately I had formatting on save set up in my editor so everything was reformatted in addition to the actual changes. I think we can go with @adrian-gierakowski version of the PR as he has cleaned this up and close this one.

---

Yes, sorry guys, I should have opened a PR by now. I’ll do later today